docs/support/add_missing_further_education_college.md

```diff
@@ -0,0 +1,139 @@
+# Add a Further Education College that is not on the service
+
+To add a Further Education College, login to the production server.
+
+```ruby
+cf login --sso
+make prod ssh
+unset RAILS_LOG_TO_STDOUT
+bundle exec rails c
+```
```

Comment:
This is covered by https://github.com/DFE-Digital/get-help-with-tech/blob/main/README.md#connecting-to-the-deployed-containers ?